components/templates/src/builtins/shortcodes/twitch.html

```diff
@@ -0,0 +1,19 @@
+<div {% if class %}class="{{class}}"{% endif %}>
+    <iframe
+        src="https://player.twitch.tv/?channel={{channel}}" 
+        frameborder="0" 
+        allowfullscreen="true"
+        height="{% if videoHeight %}{{videoHeight}}{% endif %}"
+        width="{% if videoWidth %}{{videoWidth}}{% endif %}">  
+    </iframe>
+    
+    {% if chat %}
```

Comment:
i assume there might be time where you only want the chat so I believe it should be one or the other (if chat is true, then no video)

Reply:
Well I thought that the video is always visible, I will change that.

Reply:
@Keats I'm having a problem with the if statement inside the template... the if - else if being ignored (unless I use the true/false as a string like "true"} Do you know why?

Reply:
I'm not sure what you mean, can you detail?

Reply:
Sure, I think that with your current code, the shortcode parameters must be strings, I can't use a boolean as a variable because is ignored inside the template

Reply:
ah yes it's parsed as strings :(

I'll merge that and make shortcodes take string/bool/numbers

Reply:
Actually I pushed a fix to master, can you rebase and try it?

Reply:
Should I compile from source or is updated in any package manager?

Reply:
From source, its not released yet

Reply:
Ok, I will try now